src/sidebar/components/hypothesis-app.js

```diff
@@ -119,6 +119,16 @@ function HypothesisAppController(
     scrollToView('share-dialog');
   };
 
+  this.showHelpPanel = function () {
+    if (serviceConfig(settings)) {
+      // Let the host page handle the help request.
+      bridge.call(bridgeEvents.HELP_REQUESTED);
```

Comment:
We should only call this if a callback has been provided. By default, it would be useful to show the Help panel in the client since that contains useful information such as the version number and other details.